solana-programs/start.sh

```diff
@@ -180,10 +180,10 @@ cd ../../
 cat <<EOF
 {
     "anchorProgramId": "$anchor_program_id",
-    "anchorAdminPublicKey": "$admin_keypair_publickey",
-    "anchorAdminPrivateKey": "$admin_keypair_privatekey",
-    "anchorAdminStoragePublicKey": "$admin_storage_keypair_publickey",
-    "anchorAdminStoragePrivateKey": "$admin_storage_keypair_privatekey",
+    "anchorAdminAuthorityPublicKey": "$admin_authority_keypair_publickey",
```

Comment:
is this naming ok? still some confusion about account vs address vs publickey. doesn't look like we have consistency with others either.